pkg/controller/kuberneteseventsource/doc.go

```diff
@@ -16,7 +16,9 @@ limitations under the License.
 
 // Package kuberneteseventsource implements a KubernetesEventSource controller.
 // +kubebuilder:rbac:groups=sources.eventing.knative.dev,resources=kuberneteseventsources,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=sources.eventing.knative.dev,resources=kuberneteseventsources/status,verbs=get;update;patch
 // +kubebuilder:rbac:groups=sources.eventing.knative.dev,resources=containersources,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=sources.eventing.knative.dev,resources=containersources/status,verbs=get;update;patch
```

Comment:
I don't think we need to `updateStatus` on ContainerSources in the KubernetesEventSource -- the k8s EventSource is creating the ContainerSource.